src/Standards/Generic/Tests/PHP/LowerCaseKeywordUnitTest.inc

```diff
@@ -1,25 +1,25 @@
 <?php
 // A few keywords. Obviously not valid syntax.
-class Something extends SomethingElse implements Whatever
+class Something extends SomethingElse implements Whatever {}
 abstract public private protected function whatever() {}
-const array()
-for ($var as $var) { exit; }
-if ($a and $b or $c xor $d) { die; } elseif { } else
+const array();
+foreach ($var as $var) { exit; }
+if ($a and $b or $c xor $d) { die; } elseif( $var ) { } else {}
 goto a;
 
-Class Something EXTENDS SomethingElse implementS Whatever
+Class Something EXTENDS SomethingElse implementS Whatever {}
 Abstract Public Private Protected function whatever() {}
-CONST array()
-For ($var As $var) { Exit; }
-If ($a AND $b OR $c XOR $d) { Die; } ElseIf { } Else
+CONST array();
+ForEach ($var As $var) { Exit; }
+If ($a AND $b OR $c XOR $d) { Die; } ElseIf( $var ) { } Else {}
 GOTO a;
-function (Array $a) {}
+function (Array $a) {};
 const PRIVATE;
 HttpStatus::CONTINUE;
 Function ($f) {
     Yield $f;
     Yield From fun();
-}
+};
```

Comment:
This one might have been better fixed with `$foo = Function`...

Reply:
That wouldn't fix the parse error.

Reply:
Sorry, I meant assigning to a var *as well as* the semicolon on the end - just to make it look somewhat closer to real code. That's what tripped me up with the one above.

But honestly, it's not big deal. I just started reviewing stuff because I had a few spare minutes and realised I got through a single commit before I ran out of time.

Reply:
@gsherwood I've added a new commit which addresses this for all such test code involving closures for the `Generic` sniffs.